Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface - Authors fieldtype
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I fill in "Author:" subform with:
+            | Name: | Email: |
+            |       |        |
+        When I click on the actionbar action "Publish"
+        Then I should see an element "Author" with value "This field is empty"
+
+    @javascript
+    Scenario: Fill more than one author
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I clicked on "Add another author" button
+        And I fill in "Author:" subform with:
+            | Name:   | Email:              |
+            | Author1 | author1@authors.pt  |
+            | Author2 | author2@authors.com |
+        When I click on the actionbar action "Publish"
+        Then I should see an element "Author" with value "Author1 <author1@authors.pt> Author2 <author2@authors.com>"
+
+    @javascript
+    Scenario Outline: Fill three authors then remove one then publish
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I clicked on "Add another author" button
```

Comment:
We are using EzSystems/BehatBundle Browser Context (https://github.com/ezsystems/BehatBundle/blob/master/Context/Browser/SubContext/CommonActions.php) which itself uses that when possible